src/Appwrite/Utopia/Response/Model/User.php

```diff
@@ -47,13 +47,6 @@ public function __construct()
                 'default' => new \stdClass,
                 'example' => ['theme' => 'pink', 'timezone' => 'UTC'],
             ])
-            ->addRule('roles', [
```

Comment:
Any reason this was removed?

Reply:
Yeh, it was too complected to handle the return of this output, because we only get it on runtime and don't have it in the DB. We might create a dedicated route for it or save it in the DB later.